tox.ini

```diff
@@ -51,6 +51,7 @@ BACKEND =
 # Settings defined in the top-level testenv section are automatically
 # inherited by individual environments unless overridden.
 [testenv]
+install_command = pip install --only-binary :all: --no-binary asciitree {opts} {packages}
```

Comment:
Maybe we could use environment variable instead of flags: https://pip.pypa.io/en/stable/topics/configuration/?highlight=environment#environment-variables

Reply:
That seem painful to do as you would need to set the env variable in all-but-minreq.